optimum/habana/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl.py

```diff
@@ -570,7 +1188,7 @@ def __call__(
             )
 
             # 4. Prepare timesteps
-            timesteps, num_inference_steps = retrieve_timesteps(self.scheduler, num_inference_steps, device, timesteps)
+            timesteps, num_inference_steps = retrieve_timesteps_v2(self.scheduler, num_inference_steps, device, timesteps)
```

Comment:
Can you put back the original name, and rename the `retrieve_timesteps` specific to ML perf _hpu class instead?